package.json

```
@@ -13,16 +14,19 @@
     "index.js",
     "history.js",
     "matcher.js",
-    "extra/"
+    "extra/",
+    "preact/"
   ],
   "scripts": {
     "test": "jest --verbose --coverage",
-    "size": "size-limit"
+    "size": "size-limit",
+    "prepublish": "node scripts/copy.js"
```

I would suggest dropping a script file and just inline the command here:
```suggestion
    "prepublish": "copyfiles index.js history.js matcher.js extra/** preact/"
```
Less files is better imo 😎